vespa-cloud/album-recommendation/src/test/application/tests/staging-setup/feed-without-stemming.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Feed without stemming, prior to upgrade",
+    "comment": "Assume current deployment has no stemming on the album field; run this setup",
+    "steps": [
+        {
+            "name": "feed foo",
+            "request": {
+                "cluster": "default",
+                "method": "POST",
+                "uri": "/document/v1/test/music/docid/foo",
+                "body": "../foo.json"
```

Comment:
Should we avoid dotting up the directory tree in the official examples, as it requires understanding which forefather directory will not be available?  Best practice is to make subdirectories right?  Verify the files doesn't refer to files outside the packaged tree in vespa cli?

Reply:
The point was to show how staging-setup and staging-test could reference shared resources. Hmm ...

Reply:
We could add validation. We could also reconsider. Merging now to follow other PRs.

Reply:
OK, that's a good use-case too, never mind then.